src/Sentry.Unity.Editor/SentryWindow.cs

```diff
@@ -38,7 +50,28 @@ private void OnEnable()
                 AssetDatabase.CreateAsset(Options, SentryOptionsAssetPath);
             }
 
-            EditorUtility.SetDirty(Options);
+            EditorUtility.SetDirty(Options);*/
+        }
+
+        private UnitySentryOptionsJson LoadSentryJsonConfig()
+        {
+            if (!File.Exists(SentryOptionsJsonPathFull))
+            {
+                var unitySentryOptionsJson = new UnitySentryOptionsJson
+                {
+                    Dsn = "https://94677106febe46b88b9b9ae5efd18a00@o447951.ingest.sentry.io/5439417",
+                    Enabled = true
+                };
+                var emptyOptions = JsonSerializer.Serialize(unitySentryOptionsJson, _jsonOptions);
+                File.WriteAllText(SentryOptionsJsonPathFull, emptyOptions);
+
+                // Must be called, otherwise Unity won't be able to load it with the next call. *.meta should be created for new file
+                AssetDatabase.Refresh();
+            }
+
+            // We should use `TextAsset` for read-only access in runtime. It's platform agnostic.
+            var sentryOptionsTextAsset = Resources.Load<TextAsset>(SentryOptionsJsonPath);
+            return JsonSerializer.Deserialize<UnitySentryOptionsJson>(sentryOptionsTextAsset.text, _jsonOptions)!;

```

Comment:
We could avoid  reading/deserializing here, no?

```suggestion
            return unitySentryOptionsJson;
            } else {

            // We should use `TextAsset` for read-only access in runtime. It's platform agnostic.
            var sentryOptionsTextAsset = Resources.Load<TextAsset>(SentryOptionsJsonPath);
            return JsonSerializer.Deserialize<UnitySentryOptionsJson>(sentryOptionsTextAsset.text, _jsonOptions)!;
            }

```

Reply:
You're right, missed that. Will be changed.